docs/microscope-installation-guide.md

```diff
@@ -27,11 +39,11 @@ should launch napari (may take 15 seconds on a fresh installation) with the recO
  
 ## Install and configure `Micromanager`
 
-Install `Micromanager 2.0` nightly build `20220920` (https://micro-manager.org/Micro-Manager_Nightly_Builds). 
+Install `Micromanager 2.0` nightly build `20230418` (https://micro-manager.org/Micro-Manager_Nightly_Builds). 
```

Comment:
We could give the exact link for this nightly build, https://download.micro-manager.org/nightly/2.0/Windows/MMSetup_64bit_2.0.1_20230418.exe, save folks some scrolling

Reply:
Thanks!